zeebe/protocol/ignored-changes.json

```diff
@@ -66,6 +66,24 @@
           "justification": "Block length was updated as part of generated code due to the addition of operation reference field",
           "newValue": "40",
           "oldValue": "32"
+        },
+        {
+          "ignore": true,
+          "code": "java.class.removed",
+          "old": "enum io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType",
+          "justification": "AuthorizationOwnerType is moved to security-core module"
```

Comment:
@remcowesterhoud, this is not user-facing or exported in any way, is it? Nothing changes externally for anyone, does it?

Reply:
@tmetzke Authorization records are exported to ES, but since in 8.6 we disabled all the Identity related funtionallities this refactor should be fine